utils/qemu/Makefile

```diff
@@ -298,8 +303,9 @@ CONFIGURE_ARGS +=			\
 	--enable-vhost-net		\
 	--enable-vhost-scsi		\
 	--enable-vhost-user		\
-	--enable-vhost-user-fs		\
```

Comment:
Please add a note about why this was removed

Reply:
That option was removed from qemu:
```
qemu-6.1.0$ ./configure --help | grep vhost
  vhost-net       vhost-net kernel acceleration support
  vhost-vsock     virtio sockets device support
  vhost-scsi      vhost-scsi kernel target support
  vhost-crypto    vhost-user-crypto backend support
  vhost-kernel    vhost kernel backend support
  vhost-user      vhost-user backend support
  vhost-user-blk-server    vhost-user-blk server support
  vhost-vdpa      vhost-vdpa kernel backend support
```

Reply:
It's still there, just not in the show_help output：https://github.com/qemu/qemu/blob/f9baca549e44791be0dd98de15add3d8452a8af0/configure#L1283-L1286

Maybe try again with the commands in the Makefile comment:

https://github.com/openwrt/packages/blob/f2280ae82e3ec04649639b020e4676f1d2125d13/utils/qemu/Makefile#L355-L363

Reply:
```
user@6b03bfc66c94:~/ws$ qemu_configure=build_dir/target-x86_64_musl/qemu-6.1.0/configure 
user@6b03bfc66c94:~/ws$ openwrt_makefile=feeds/packages/utils/qemu/Makefile 
user@6b03bfc66c94:~/ws$ for arg in $(grep -E '^\s*--disable-[^)]+\)' "$qemu_configure" | cut -f1 -d')'); do grep -qE "(--enable|--disable)${arg#--disable}" "$openwrt_makefile" || echo "$arg"; done
--disable-virtiofsd
--disable-vnc
--disable-gettext
--disable-block-drv-whitelist-in-tools
--disable-tsan
--disable-vnc-sasl
--disable-vnc-jpeg
--disable-vnc-png
--disable-slirp
--disable-nvmm
--disable-malloc-trim
--disable-spice
--disable-spice-protocol
--disable-pie
--disable-safe-stack
--disable-cfi
--disable-cfi-debug
--disable-fdt
--disable-bpf
--disable-blobs
--disable-vhost-user-fs
--disable-u2f
--disable-libusb
--disable-zlib-test
--disable-avx2
--disable-avx512f
--disable-virtio-blk-data-plane|--enable-virtio-blk-data-plane
--disable-git-update
--disable-fuzzing
--disable-rng-none
--disable-keyring
--disable-libdaxctl
--disable-fuse
--disable-fuse-lseek
--disable-multiprocess
--disable-gio
--disable-slirp-smbd
```

Reply:
Checked with vhost-user-fs - all ok.